### ISSUE TITLE
Crash when loading 100% world Digitalxero seed

### ISSUE THREAD

**Issue by djgilcrease** [thirite/ChildrenAndPregnancy#29]
After experimenting a while I have found seeds that work and some that don't and with the stack trace it seem like leaders spawn as a child causes the crash.

```
========== OUTPUTING STACK TRACE ==================

0x00B7AE48 (RimWorldWin) Texture2D::AwakeFromLoad
0x00C3AD82 (RimWorldWin) RegisterAllowNameConversionInDerivedTypes
0x00C3AF6D (RimWorldWin) RegisterAllowNameConversionInDerivedTypes
0x00C41D0D (RimWorldWin) RegisterAllowNameConversionInDerivedTypes
0x00C41DF7 (RimWorldWin) RegisterAllowNameConversionInDerivedTypes
0x009F809F (RimWorldWin) ReadObjectFromPersistentManager
0x00F1C988 (RimWorldWin) MeshFilter::VirtualRedirectTransfer
0x00CCB937 (RimWorldWin) AnimationCurveTpl<float>::Sort
0x06689943 (Mono JIT Code) (wrapper managed-to-native) UnityEngine.Resources:LoadAll (string,System.Type)
0x066898D4 (Mono JIT Code) UnityEngine.Resources:LoadAll<object> (string)
0x2D462982 (Mono JIT Code) Verse.GraphicDatabaseUtility/<GraphicNamesInFolder>c__Iterator22D:MoveNext ()
0x2D4627AB (Mono JIT Code) Verse.GraphicDatabaseHeadRecords:BuildDatabaseIfNecessary ()
0x2D462400 (Mono JIT Code) Verse.GraphicDatabaseHeadRecords:GetHeadRandom (Verse.Gender,UnityEngine.Color,Verse.CrownType)
0x2D462395 (Mono JIT Code) RimWorld.Pawn_StoryTracker:get_HeadGraphicPath ()
0x05355792 (Mono JIT Code) (wrapper dynamic-method) Verse.PawnGraphicSet:ResolveAllGraphics_Patch1 (object)
0x2D460FB7 (Mono JIT Code) RimWorldChildren.Hediff_Baby:GrowUpTo (int,bool)
0x2D4548E6 (Mono JIT Code) RimWorldChildren.PawnGenerate_Patch:_GeneratePawn (Verse.PawnGenerationRequest&,Verse.Pawn&)
0x0535713D (Mono JIT Code) (wrapper dynamic-method) Verse.PawnGenerator:GeneratePawn_Patch1 (Verse.PawnGenerationRequest)
0x2D43D44E (Mono JIT Code) Verse.PawnGenerator:GeneratePawn (Verse.PawnKindDef,RimWorld.Faction)
0x2D43AA98 (Mono JIT Code) RimWorld.Faction:GenerateNewLeader ()
0x2D43A3B7 (Mono JIT Code) RimWorld.FactionGenerator:NewGeneratedFaction (RimWorld.FactionDef)
0x2D439CE3 (Mono JIT Code) RimWorld.FactionGenerator:GenerateFactionsIntoWorld ()
0x2D439BD7 (Mono JIT Code) RimWorld.Planet.WorldGenStep_Factions:GenerateFresh (string)
0x2D424CE5 (Mono JIT Code) RimWorld.Planet.WorldGenerator:GenerateWorld (single,string,RimWorld.Planet.OverallRainfall,RimWorld.Planet.OverallTemperature)
0x2D424A99 (Mono JIT Code) RimWorld.Page_CreateWorldParams:<CanDoNext>m__615 ()
0x0669C711 (Mono JIT Code) Verse.LongEventHandler:RunEventFromAnotherThread (System.Action)
0x0669C6D0 (Mono JIT Code) Verse.LongEventHandler:<UpdateCurrentAsynchronousEvent>m__84C ()
0x0662F541 (Mono JIT Code) (wrapper runtime-invoke) object:runtime_invoke_void__this__ (object,intptr,intptr,intptr)
0x100F1440 (mono) mono_set_defaults
0x1005D7EE (mono) mono_runtime_invoke
0x1005DC0A (mono) mono_runtime_delegate_invoke
0x1007EFBD (mono) mono_thread_interruption_request_flag
0x1010C4F4 (mono) unity_mono_method_is_generic
0x75548744 (KERNEL32) BaseThreadInitThunk
0x7753582D (ntdll) RtlGetAppContainerNamedObjectPath

========== END OF STACKTRACE ===========

**** Crash! ****
```

**Comment by thirite** [comment]
Thanks for the diagnosis! I wonder if this is the root cause of the issue all along.

**Comment by thirite** [comment]
Fixed in 4dcc220c723d844894437c8b860f58e3eeb3e909
